test/parallel/test-vm-debug-context-token.js

```diff
@@ -0,0 +1,12 @@
+var common = require('../common');
+var vm = require('vm');
+var assert = require('assert');
+
+var proto = vm.runInDebugContext('DebugCommandProcessor.prototype');
+proto.prop = true;
+assert.equal(proto.prop, true);
+proto = vm.runInDebugContext('DebugCommandProcessor.prototype');
+assert.equal(proto.prop, true);
+proto = vm.runInDebugContext('DebugCommandProcessor.prototype.prop = true;' +
```

Comment:
My understanding of this line was it was testing that if you set it inside the vm, instead of outside, it still works. So it might be good to leave it in but use a different value.